autofs-5.1.1 - update libtirpc workaround for new soname

From: Ian Kent <ikent@redhat.com>

When pre-opening the libtirpc shared library also check for revision
3 of the library.
---
 CHANGELOG          |    4 ++++
 daemon/automount.c |    2 ++
 2 files changed, 6 insertions(+)

diff --git a/CHANGELOG b/CHANGELOG
index dbfbe48..688d485 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -1,3 +1,7 @@
+??/??/2015 autofs-5.1.2
+=======================
+- update libtirpc workaround for new soname.
+
 21/04/2015 autofs-5.1.1
 =======================
 - fix compile error in defaults.c.
diff --git a/daemon/automount.c b/daemon/automount.c
index 229cb1a..c81b4af 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -2393,6 +2393,8 @@ int main(int argc, char *argv[])
 	void *dh_tirpc = dlopen("libtirpc.so", RTLD_NOW);
 	if (!dh_tirpc)
 		dh_tirpc = dlopen("libtirpc.so.1", RTLD_NOW);
+	if (!dh_tirpc)
+		dh_tirpc = dlopen("libtirpc.so.3", RTLD_NOW);
 #endif
 
 	if (!master_read_master(master_list, age, 0)) {
